packages/core/interfaces/index.ts

```diff
@@ -108,4 +108,16 @@ export interface UseContent<CONTENT, SEARCH> {
   search: SEARCH,
   loading: boolean,
   error: any
-}
\ No newline at end of file
+}
+
+export interface UiMediaGalleryItemUrl {
+  url: string
+}
+export interface UiMediaGalleryItem {
+  small: UiMediaGalleryItemUrl
```

Comment:
Wouldn't it be misleading in autosuggestions? User won't see if thats a string or something else. While typing response makes sense to em renaming strings to other properties desn't and is confusing
Again @patzick  for validation

Reply:
Here is an example of confusion - user even doesnt know that its a string 
![image](https://user-images.githubusercontent.com/15185752/71353770-5b867e00-257a-11ea-9122-cbef24b83e38.png)